### DISCUSSION TITLE
geojson data not displayed on plotly correctly

### DISCUSSION THREAD

**Discussion by ghost** [vanna-ai/vanna#642] [Q&A]
Hello, 

I have some problems dealing with displaying map data and i would like to verify how is that possible.
i typed this question **plot all polygones on a map from geometry data in shapes**

**Error logs/Screenshots**
![image](https://github.com/user-attachments/assets/e5237425-e30e-409c-ad91-6267d292d715)
![image](https://github.com/user-attachments/assets/d2119c83-e64c-47da-9fd6-6118c24ab31e)

**Desktop**
 - OS: Ubuntu
 - Version: 22.04
 - Python: 3.12
 - Vanna: latest
 - Connector: postgres/postgis

**Comment by Elienvalleau** [comment]
Hello, did you find a solution ?

**Comment by lucasseidor33** [comment]
Hello, did you find a solution ?
